consensus/tendermint/action.go

```diff
@@ -14,10 +10,10 @@ type BroadcastPrecommit[H types.Hash, A types.Addr] types.Precommit[H, A]
 
 type ScheduleTimeout types.Timeout
 
-func (a *BroadcastProposal[V, H, A]) isTendermintAction() {}
+func (a *BroadcastProposal[V, H, A]) IsTendermintAction() {}
```

Comment:
We should move all of them. `isTendermintAction` needs to be private in order to seal the interface.